modules/ROOT/pages/import-api-specification-maven.adoc

```diff
@@ -0,0 +1,63 @@
+= Importing an API Specification from Maven
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Studio allows you to import an API specification in a Maven repository into a new Mule Project, or an existing one.
+
+This feature is possible through API Sync. If you are using an older version of Mule Runtime (4.1.3 and below), or if you don't want to use this new design paradigm and would still prefer to import an API specification that's not in Exchange, follow the xref:studio::import-api-specification-design-center.adoc[Importing an API Specification from a Local File] steps.
+
+== Importing an API From Maven Into a New Project
+
```

Comment:
DITTO consider text transition between title and first step if there is something useful to add about why our audience might want to choose this method over creating a new project. GLOBAL in these topics.